source/_integrations/sensehat.markdown

```diff
@@ -87,13 +87,13 @@ sensor:
   - platform: template
     sensors:
       sensehat_temperature:
-        value_template: '{{ states('sensor.temperature') | round(1) }}'
+        value_template: '{{ states("sensor.temperature") | round(1) }}'
```

Comment:
This is not according to our documentation standards. See: <https://developers.home-assistant.io/docs/documenting/standards#templates>

```suggestion
        value_template: "{{ states('sensor.temperature') | round(1) }}"
```

Please adjust the other cases below as well.

Reply:
But this means the final user cannot do cut&paste of our example, which is confusing.

Reply:
Yes they can.